lib/wibox/init.lua

```diff
@@ -357,7 +371,7 @@ local function new(args)
     -- If a value is not found, look in the drawin
     setmetatable(ret, {
         __index = function(self, k)
-            if wibox["get_"..k] then
+            if rawget(self, "get_"..k) then
```

Comment:
Huh? I'll just ask: Is this on purpose and does it really belong into this commit? Further, I'll assume that the answer is "yes" and won't try to understand the details.